app/validators/sections/section_validator.py

```diff
@@ -93,9 +101,14 @@ def validate_routing(self, schema_element, group):
                 )
             self.errors += routing_validator.validate()
         if "skip_conditions" in schema_element:
-            self.validate_skip_conditions(
-                schema_element["skip_conditions"], schema_element["id"]
-            )
+            if isinstance(schema_element["skip_conditions"], list):
+                self.validate_skip_conditions(
+                    schema_element["skip_conditions"], schema_element["id"]
+                )
+            elif isinstance(schema_element["skip_conditions"], dict):
+                self.validate_new_skip_conditions(
+                    [schema_element["skip_conditions"]], schema_element["id"]
```

Comment:
How come this is passed as a list?. You just need to pass the single when condition.